models/deepseek/v4/hc_pre.py

```
@@ -49,7 +49,11 @@
 # tiling
 T_TILE = 16  # unified row-tile for the fused spmd
 RMS_K_TILE = 128
-LINEAR_K_TILE = 128
+# 256 (not 128/512): the matmul K-reduction is a serial matmul_acc dependency
+# chain (HC_DIM/LINEAR_K_TILE steps), each carrying fixed MTE/sync overhead.
+# Doubling the K-tile halves the chain length (−~29% on the matmul in the
```

Ruff (0.15.18)

[warning] Comment contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)